content/concepts/the-rippled-server/rippled-server-modes.md

```diff
@@ -2,61 +2,83 @@
 
 
-## Reasons to Run a Validator
+### Validators
 
-The robustness of the XRP Ledger depends on an interconnected web of validators who each trust a few other validators _not to collude_. The more operators with different interests there are who run validators, the more certain each member of the network can be that it continues to run impartially. If you or your organization relies on the XRP Ledger, it is in your interest to contribute to the consensus process.
+The robustness of the XRP Ledger depends on an interconnected web of _validators_ who each trust some other validators _not to collude_. In addition to processing each transaction and calculating ledger state just like other network mode servers, validators participate actively in the [consensus protocol](consensus.html). If you or your organization relies on the XRP Ledger, it is in your interest to contribute to the consensus process by running _one_ server as a validator.
 
-Not all `rippled` servers need to be validators: trusting more servers from the same operator does not offer better protection against collusion. An organization might run validators in multiple regions for redundancy in case of natural disasters and other emergencies.
+Validation uses only a small amount of computing resources, but there is not much benefit to a single entity or organization running multiple validators because doing so does not provide more protections against collusion. Each validator identifies itself with a unique cryptographic key pair that must be carefully managed; multiple validators must not share a key pair. For these reasons, validation is disabled by default.
 
-If your organization runs a validating server, you may also run one or more stock servers, to balance the computing load of API access, or as a proxy between your validation server and the outside network.
+You can safely enable validation on a server that is also used for other purposes; this type of configuration is called an _all-purpose server_. Alternatively, you can run a _dedicated validator_ that does not perform other tasks, possibly in a [cluster](clustering.html) with other network mode `rippled` servers.
 
 For more information about running a validator, see [Run `rippled` as a Validator](run-rippled-as-a-validator.html).
 
 
+## Reporting Mode
+[New in: rippled 1.7.0][]
+
+Reporting mode is specialized mode for serving API requests more efficiently. In this mode, the server connects to a separate Network Mode `rippled` server to get the latest validated ledger data, then loads that data into a relational database ([PostgreSQL](https://www.postgresql.org/)). The reporting mode server does not directly participate in the peer-to-peer network, though it can forward requests such as transaction submission to the network mode server it uses.
```

Comment:
> connects to a separate Network Mode `rippled` server to get the latest validated ledger data,

How about:

> connects to a separate rippled` server running in Network Mode `to get the latest validated ledger data,

Reply:
Also "Network Mode" is inconsistently capitalized in this paragraph. For all modes, we should stick to one or the other.

Reply:
Changed to "P2P Mode" (capitalization consistent) according to the preference of the `rippled` team and for consistency with the code & example config file.